mlflow/server/handlers.py

```diff
@@ -477,16 +477,8 @@ def _get_request_json(flask_request=request):
 
 
 def _get_request_message(request_message, flask_request=request, schema=None):
-    from querystring_parser import parser
-
     if flask_request.method == "GET" and len(flask_request.query_string) > 0:
-        # This is a hack to make arrays of length 1 work with the parser.
-        # for example experiment_ids%5B%5D=0 should be parsed to {experiment_ids: [0]}
-        # but it gets parsed to {experiment_ids: 0}
-        # but it doesn't. However, experiment_ids%5B0%5D=0 will get parsed to the right
-        # result.
```

Comment:
Need to check whether MLflow UI sends requests with this format.

Reply:
Hmm, one weird example is for the `get-history-bulk-interval` API (fetching metrics for charts)

The query string looks like:
```
http://localhost:3000/ajax-api/2.0/mlflow/metrics/get-history-bulk-interval?run_ids=2a1a38961b4c4bb684dfd084451daeab&run_ids=9c6f757ffdaa42afafb948d14d191757&metric_key=precision&max_results=320
```

Reply:
`get-history-bulk-interval` is already using `args.to_dict(flat=False)`:

https://github.com/mlflow/mlflow/blob/0c26482f023078d723aa13cc604ace7335e8b9a1/mlflow/server/handlers.py#L1103-L1106